akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -525,7 +532,14 @@ object ShardCoordinator {
     }
     var remaining = regions
 
-    log.debug("Rebalance [{}] from region [{}]", shard, regions)
+    if (isRebalance)
+      log.debug("Rebalance [{}] from region [{}]", shard, regions)
```

Comment:
In https://github.com/akka/akka/pull/29590 I tried to get away from logging full Set of regions, and instead use the size. Maybe this PR isn't rebased?

Could you also fix my parameter order mistake? https://github.com/akka/akka/pull/29590/files#r487298916